src/index.ts

```diff
@@ -215,7 +215,7 @@ function teenyRequest(
   const multipart = reqOpts.multipart as RequestPart[];
   if (reqOpts.multipart && multipart.length === 2) {
     if (!callback) {
-      console.log('Error, multipart without callback not implemented.');
+      console.error('multipart without callback not implemented.');
```

Comment:
Shouldn't we throw an exception or something in this case?

Reply:
I have always been under the impression that trying to log like this is not awesome for libraries.  The only case (I can think of) where the callback wasn't defined is if the user was expecting to get a stream back.  In that case ... throwing feels correct, but also highly scary as folks are unlikely to catch it.

Reply:
my gut was an exception should be thrown here too, but I'm also feeling like we're down a rabbit hole, I want us to be back to a working version of teeny-request in @google-cloud/storage as soon as possible.

Let's log some tickets for this?